mne/preprocessing/maxwell.py

```diff
@@ -1896,6 +1900,29 @@ def find_bad_channels_maxwell(
     flat_chs : list
         List of MEG channels that were detected as being flat in at least
         ``min_count`` segments.
+    scores : dict
+        A dictionary with information produced by the scoring algorithms.
+        Only returned when ``return_scores`` is ``True`` and contains the
+        following keys:
+
+        - ``bin_edges`` : ndarray, shape (n_windows + 1,)
+
+          The window boundaries (in seconds) used to calculate the scores.
+
+        - ``scores_flat`` : ndarray, shape (n_meg, n_windows)
+
+          The scores for testing whether MEG channels are flat.
+
+        - ``limits_flat`` : ndarray, shape (n_meg, n_windows)
+
+          The thresholds above which a score classified a segment as "flat".
+
+        - ``scores_noisy`` : ndarray, shape (n_meg, n_windows)
```

Comment:
insert empty line to be consistent. Or remove it everywhere to avoid a too long docstring

Reply:
I've added an empty line for now.

I'm not sure what's the best approach, also considering that I want a nice formatting in the end :) @drammock, do you have any suggestions here?